src/main/java/helper/SuccessResponseGenerator.java

```diff
@@ -94,20 +88,7 @@
      */
     public static Map<String, Object> getSuccessResponseForGetMessage(Message message, Attachment attachment, long userId) {
 
-        Map<String, Object> responseBody = new LinkedHashMap<String, Object>();
-        responseBody.put("MessageId", message.getMessageId());
-        responseBody.put("ChatId", message.getChatId());
-        
-        if (message.getSenderId() == userId) {
-            responseBody.put("SentByCurrentUser", true);
-        } else {
-            responseBody.put("SentByCurrentUser", false);
-        }
-
-        responseBody.put("SentTs", message.getSentTs());
-        responseBody.put("ReceivedTs", message.getReceivedTs());
-        responseBody.put("CreationTs", message.getCreationTs());
-        responseBody.put("TextContent", message.getTextContent());
+        Map<String, Object> responseBody = getSuccessResponseForGetMessage(Message message, long userId);
```

Comment:
Error?
You copy pasted the data types too.

Reply:
Done